example_project/mappers/Baseline.rb

```diff
@@ -1073,6 +1073,236 @@ def time_mapping(time)
+          'AZ': 'AZNMc',
+          'ME': 'NEWEc',
+          'AL': 'SRSOc',
+          'MD': 'RFCWc',
+          'NH': 'NEWEc',
+          'MA': 'NEWEc',
+          'ND': 'MROWc',
+          'NV': 'NWPPc',
+          'CT': 'NEWEc',
+          'DE': 'RFCEc',
+          'RI': 'NEWEc'}
+
+          #California, Carolinas, Central, Florida, Mid-Atlantic, Midwest, New England, New York, Northwest, Rocky Mountains, Southeast, Southwest, Tennessee, and Texas
+          hourly_historical_mapping_hash = 
+          {'FL': 'Florida',
+          'MS': 'SRMVc',
+          'NE': 'Midwest',
+          'OR': '',
+          'CA': '',
+          'VA': '',
+          'AR': '',
+          'TX': '',
+          'OH': '',
+          'UT': '',
+          'MT': '',
+          'TN': '',
+          'ID': '',
+          'WI': '',
+          'WV': '',
+          'NC': '',
+          'LA': '',
+          'IL': '',
+          'OK': '',
+          'IA': '',
+          'WA': '',
+          'SD': '',
+          'MN': '',
+          'KY': '',
+          'MI': '',
+          'KS': '',
+          'NJ': '',
+          'NY': '',
+          'IN': '',
+          'VT': '',
+          'NM': '',
+          'WY': '',
+          'GA': '',
+          'MO': '',
+          'DC': '',
+          'SC': '',
+          'PA': '',
+          'CO': '',
+          'AZ': '',
+          'ME': '',
+          'AL': '',
+          'MD': '',
+          'NH': '',
+          'MA': '',
+          'ND': '',
+          'NV': '',
+          'CT': '',
+          'DE': '',
+          'RI': ''}
+
+          annual_historical_mapping_hash = 
+          {'FL': 'FRCC',
+          'MS': 'SRMV',
+          'NE': 'MROW',
+          'OR': 'NWPP',
+          'CA': 'CAMX',
+          'VA': 'RFCW',
+          'AR': 'SRMV',
+          'TX': 'ERCT',
+          'OH': 'RFCW',
+          'UT': 'NWPP',
+          'MT': 'NWPP',
+          'TN': 'SRTV',
+          'ID': 'NWPP',
+          'WI': 'MROW',
+          'WV': 'RFCW',
+          'NC': 'SRVC',
+          'LA': 'SRMV',
+          'IL': 'RFCW',
+          'OK': 'SPSO',
+          'IA': 'MROW',
+          'WA': 'NWPP',
+          'SD': 'MROW',
+          'MN': 'MROW',
+          'KY': 'SRTV',
+          'MI': 'RFCM',
+          'KS': 'SPNO',
+          'NJ': 'RFCE',
+          'NY': 'NYST',
+          'IN': 'RFCW',
+          'VT': 'NEWE',
+          'NM': 'AZNM',
+          'WY': 'NWPP',
+          'GA': 'SRSO',
+          'MO': 'SRMW',
+          'DC': 'RFCE',
+          'SC': 'SRVC',
+          'PA': 'RFCW',
+          'CO': 'RMPA',
+          'AZ': 'AZNM',
+          'ME': 'NEWE',
+          'AL': 'SRSO',
+          'MD': 'RFCW',
+          'NH': 'NEWE',
+          'MA': 'NEWE',
+          'ND': 'MROW',
+          'NV': 'NWPP',
+          'CT': 'NEWE',
+          'DE': 'RFCE',
+          'RI': 'NEWE'}
+
+          #get the state from weather file
+          state = feature.weather_filename.split('_', 1)
+          
+          #apply emissions inputs based on the state
```

Comment:
if you can get some of these from the weather file state, do we still need them in the geojson file?

Reply:
addressed